enhancements/helm3/console.md

```diff
@@ -0,0 +1,144 @@
+---
+title: helm-charts-in-developer-catalog
+authors:
+  - "@sbose78"
+  - "@pedjak"
+reviewers:
+  - TBD
+approvers:
+  - TBD
```

Comment:
if the change is to core openshift api, please add an openshift api approver here.

recommend @deads2k @bparees and myself.

Reply:
+ @spadgett